### PR TITLE
Paperwork update

### PR DESCRIPTION
**Pull request by symphorien**
###### Motivation for this change

paperwork now has a desktop file!

###### Things done

paperwork, paperwork-backend: 1.2.4 -> 1.3.0
libinsane2: init at 1.0.2
pyinsane2: remove, superseeded by libinsane2 (there is no other reverse dependency in nixpkgs)

I can't run nix-review because natsort does not build on master.

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by symphorien** [comment]
cc @aszlig

File: pkgs/applications/office/paperwork/default.nix [hunk 1]
```diff
@@ -46,9 +54,27 @@ python3Packages.buildPythonApplication rec {
     paths = lib.collect lib.isDerivation aspellDicts;
   }}/lib/aspell";
 
+  postInstall = ''
+    # paperwork-shell needs to be re-wrapped with access to paperwork
+    cp ${python3Packages.paperwork-backend}/bin/.paperwork-shell-wrapped $out/bin/paperwork-shell
+    # install desktop files and icons
+    # paperwork-shell install copies everything to $HOME/.local/share instead of $out/share
+    mkdir -p home/.local
```
Comment (aszlig) [inline pkgs/applications/office/paperwork/default.nix]: Hm, can't this be directly patched instead?

**Comment by jonringer** [comment]
please fix up the commit history

**Comment by jonringer** [comment]
should be roughly a commit per package change

File: pkgs/applications/office/paperwork/default.nix (hunk 1 above)
Comment (symphorien) [reply pkgs/applications/office/paperwork/default.nix]: Patching is not better, because the user is supposed to be able to call paperwork-shell install.

**Comment by symphorien** [comment]
About rewriting the history: my git-fu is not good enough. In any case, I ticked the checkbox "Allow edits from maintainers." so if it is important to you, feel free to edit this PR to your liking.

**Comment by symphorien** [comment]
ping

**Comment by jonringer** [comment]
`git rebase -i` is your friend

```
git rebase -i HEAD~12
# clump related commits together, the old commit is at the top, youngest at the bottom
# for commits can be "rolled-up" to the previous, edit pick to be squash
# save buffer
git push <fork> <branch> --force
```

**Comment by symphorien** [comment]
No time to dedicate to fighting with git.

**Pull request closed without merging**